.gitignore

```diff
@@ -1,16 +1,12 @@
 .bundle/
 log/*.log
 pkg/
-test/dummy/db/*.sqlite3
-test/dummy/db/*.sqlite3-journal
-test/dummy/log/*.log
-test/dummy/tmp/
-test/dummy/.sass-cache
-test/dummy*/db/*.sqlite3
-test/dummy*/db/*.sqlite3-journal
-test/dummy*/log/*.log
-test/dummy*/tmp/
-test/dummy*/.sass-cache
+test/rails_apps/*/.sass-cache
+test/rails_apps/*/db/*.sqlite3
+test/rails_apps/*/db/*.sqlite3-journal
+test/rails_apps/*/log/*.log
+test/rails_apps/*/tmp/
+test/rails_apps/*/.sass-cache
```

Comment:
I ended up moving all the rails apps to be named consistently which helped clean up things in a few spots including here.